components/inputs/input-date-time.js

```diff
@@ -308,7 +308,12 @@ class InputDateTime extends FocusMixin(LabelledMixin(SkeletonMixin(FormElementMi
 		if (!this.shadowRoot) return;
 		const dateInput = this.shadowRoot.querySelector('d2l-input-date');
 		const timeInput = this.shadowRoot.querySelector('d2l-input-time');
-		const errors = await Promise.all([dateInput.validate(), timeInput.validate(), super.validate()]);
+
+		const errors = await Promise.all([
+			dateInput.validate(),
+			timeInput ? timeInput.validate() : Promise.resolve([]),
```

Comment:
If `timeInput` is `null` then we don't need to validate it. It is `null` because the user has not selected a date. If the input is required, the date validation will fail. Once the user selects a date, `timeInput` will be rendered and we will validate as usual.